client/clientd/src/main.rs

```diff
@@ -22,8 +22,20 @@ use tracing::{info, Level};
 use tracing_subscriber::EnvFilter;
 
 #[derive(Parser)]
-struct Config {
-    workdir: PathBuf,
+struct Cli {
+    #[clap(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand)]
+enum Command {
+    /// Print the latest git commit hash this bin. was build with
+    VersionHash,
+    // Stard clientd with configuration
```

Comment:
starD? Seems like you use it everywhere. Is it a typo, or am I missing something?

Reply:
I have no idea how I came to the conclusion that starD is the correct spelling haha I was like wait is it start or stard and even googled it I think, idk xd